examples/inlineHello_cluster_cron.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: activemonitor.keikoproj.io/v1alpha1
```

Comment:
certainly a great idea to include a scheduler/cron based healthcheck example as a part of this feature work :)